Formula/bazaar.rb

```diff
@@ -16,7 +16,8 @@ class Bazaar < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:    "cb1c0c8b5f19abef4043195d8cbd19f363a78581596de1ddcc763621964335b3"
   end
 
-  deprecate! date: "2021-08-19", because: "is not supported. Check out `breezy` instead"
+  # Commented out while this formula still has dependents.
+  # deprecate! date: "2021-08-19", because: "is not supported. Check out `breezy` instead"
```

Comment:
Not sure if this has any non-deprecated dependents.

Reply:
`brew` seems to think `schroedinger` depends on `bazaar`:
```
schroedinger:
  * Dependency 'bazaar' is deprecated but has un-deprecated dependents. Either
    un-deprecate 'bazaar' or deprecate all of its dependents.
```

Reply:
I guess from head build (`url "lp:schroedinger", using: :bzr`). Can we switch to `breezy`?

Reply:
`breezy` is supposed to be a drop-in replacement, right? If so we probably should. Let me look at it. But I'd rather do it as a follow-up.

Reply:
Homebrew/brew#13617

We probably need it to land in a tagged release before properly deprecating `bazaar`, though.